src/crud/collection.py

```
@@ -0,0 +1,65 @@
+        select(models.Collection)
+        .where(models.Collection.workspace_name == workspace_name)
+        .where(models.Collection.name == collection_name)
+    )
+    if peer_name:
+        stmt = stmt.where(models.Collection.peer_name == peer_name)
+    result = await db.execute(stmt)
+    collection = result.scalar_one_or_none()
+    if collection is None:
+        raise ResourceNotFoundException(
+            "Collection not found or does not belong to peer"
+        )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)